js/packages/teams-ai/src/Application.spec.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-named-as-default-member */
```

Comment:
We can use the "override" key instead for .spec.ts files instead of adding it per file
https://stackoverflow.com/a/50101673

This would be cleaner + we won't need to remember to add it when we add another test file!

Reply:
You're the best!